setup.py

```diff
@@ -55,6 +55,8 @@ def run(self):
 
 install_requires = [
     "alembic>=0.6.6",
+    "amqp>=1.4.5,<2.0",
+    "anyjson>=0.3.3",
```

Comment:
Even with the newest PU and after reinstall I get this:

 inveniomanage runserverTraceback (most recent call last):
  File "/home/wojciech/.virtualenvs/scoap3-next/bin/inveniomanage", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/home/wojciech/.virtualenvs/scoap3-next/local/lib/python2.7/site-packages/pkg_resources.py", line 2716, in <module>
    working_set.require(__requires__)
  File "/home/wojciech/.virtualenvs/scoap3-next/local/lib/python2.7/site-packages/pkg_resources.py", line 685, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/wojciech/.virtualenvs/scoap3-next/local/lib/python2.7/site-packages/pkg_resources.py", line 588, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: anyjson>=0.3.3